docs/services/data/_index.md

```diff
@@ -150,21 +150,96 @@ err := data.Sync(context.Background(), nil)
 {{% /tab %}}
 {{< /tabs >}}
 
+### TabularDataBySQL
+
+{{% alert title="Important" color="tip" %}}
+
+This method is not yet available in the Viam Python SDK.
+
+{{% /alert %}}
+
+Query tabular data that has been synced to the Viam app using {{< glossary_tooltip term_id="sql" text="SQL" >}}.
+
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `sql_query` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The SQL `SELECT` statement to run against your synced tabular data.
+  Tabular data is held in a database named `sensorData` and a table named `readings`, so you your query should `SELECT` from `"readings"` or `"sensorData.readings"`.
+
+**Returns:**
+
+- (object): A struct containing the query results.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/datamanager).
+
+```go {class="line-numbers linkable-line-numbers"}
+// code example
```

Comment:
Q / TODO: Do you have a code sample somewhere I can use? Not seeing this method in the Go SDK docs. If not I can come up with something.

Reply:
Linked the TypeScript SDK docs, as these are not available in Go

Reply:
Thank you!